map_org_names.yaml

```diff
@@ -226,7 +226,7 @@ mappings:
   - ['^[[:space:]]*pagero[[:space:]]*$', 'Pagero AB']
   - ['^[[:space:]]*palantir[[:space:]]*$', 'Palantir Technologies']
   - ['^[[:space:]]*palo[[:space:]]*alto[[:space:]]*network[[:space:]]*$', 'Palo Alto Networks']
-  - ['^[[:space:]]*(pantheon\\.?tech|pantheon\\.?tech[[:space:]]*s\\.?r\\.?o\\.?|pantheon)[[:space:]]*$', 'Pantheon Technologies']
+  - ['^[[:space:]]*(pantheon\\.?tech|pantheon\\.?tech[[:space:]]*s\\.?r\\.?o\\.?|pantheon)[[:space:]]*$', 'PANTHEON.tech']
```

Comment:
Not sure if this will work, since we want to replace ```Pantheon Technologies``` with ```PANTHEON.tech``` and the regular expression will try to replace pantheon.tech.

Reply:
Yes, since we are updating the RE manually, let's create such that will not hit the final replacement name.

Reply:
@lukaszgryglicki how will this look, my understanding is
```
['^[[:space:]]*pantheon[[:space:]]technologies[[:space:]]*$', 'PANTHEON.tech']```
Since we have to change ```Pantheon Technologies``` to ```PANTHEON.tech```

Reply:
I would use: `['^[[:space:]]*(pantheon[[:space:]]*technologies|pantheon)[[:space:]]*$', 'PANTHEON.tech']`
Just add `*` after `[[:space:]]` between `pantheon` and `technologies`.
Or maybe even: `['^[[:space:]]*(pantheon[[:space:]](tech|technologies)|pantheon)[[:space:]]*$', 'PANTHEON.tech']` (`technoligies` -> `tech`).
It also seems that you want to skip the `s\\.?r\\.?o\\.?` branch?

Reply:
Actually this is best to just try it on the SH DB, like this:
```
select * from organizations where name regexp '^[[:space:]]*(pantheon\\.?tech|pantheon\\.?tech[[:space:]]*s\\.?r\\.?o\\.?|pantheon)[[:space:]]*$';
```